packages/hub/routes/scheduled-payments.ts

```diff
@@ -215,6 +232,42 @@ export default class ScheduledPaymentsRoute {
 
     ctx.type = 'application/vnd.api+json';
   }
+
+  async cancel(ctx: Koa.Context) {
```

Comment:
Is this wired up somewhere? I’d expect a corresponding change in `api-router`

Reply:
Oops, forgot to remove this endpoint. We're only using PATCH in this case.